artix7/tests/buttons/buttons.pcf

```diff
@@ -0,0 +1,17 @@
+set_io in[0] V17
```

Comment:
Can you add comments to this file?

I assume the top set is the buttons on the basys and the bottom set are the LEDs on the basys?